test/e2e_federation/upgrade.go

```diff
@@ -25,7 +25,7 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-var upgradeTests = []upgrades.Test{}
+var upgradeTests = upgrades.SimpleUpgradeTests()
```

Comment:
YAGNI - whoever ends up adding more types can figure out how best to add to the list of upgrade tests.

Also, I'm calling it 'simple' for now, but I anticipate being able to perform upgrade tests generically on all resources by the end of 1.7.